scss/_layouts_application.scss

```diff
@@ -235,7 +235,7 @@ $application-layout--side-nav-width-expanded: 15rem !default;
   }
 
   .l-aside {
-    @include vf-transition($property: #{transform, box-shadow}, $duration: snap);
+    @include vf-transition($property: #{transform, box-shadow, visibility}, $duration: snap);
```

Comment:
Is visibility a prop that can be transitioned?

Reply:
Yes, it can: https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#interpolation

Reply:
Try the demo with visibility transition and without to see for yourself 😄

Reply:
https://vanilla-framework-4628.demos.haus/docs/layouts/application